subprojects/docs/src/docs/userguide/jvm/building_java_projects.adoc

```diff
@@ -276,20 +287,19 @@ To use Java 6 or Java 7, the following tasks need to be configured:
 * `Javadoc` task to use the correct `javadoc` executable
 * `Test` and the `JavaExec` task to use the correct `java` executable.
 
-The following sample shows how the `build.gradle` needs to be adjusted. In order to be able to make the build machine-independent, the location of the old Java home and target version should be configured in `GRADLE_USER_HOME/gradle.properties` footnote:[For more details on `gradle.properties` see <<build_environment.adoc#sec:gradle_configuration_properties,Gradle configuration properties>> ] in the user's home directory on each developer machine, as shown in the example.
+With the usage of Java toolchains, this can be done as follows:
 
 ==== Example: Configure Java 7 build
-[source,properties]
-.gradle.properties
-----
-include::{snippetsPath}/java/crossCompilation/groovy/gradle.properties[]
-----
 
 ====
 include::sample[dir="snippets/java/crossCompilation/groovy",files="build.gradle[tags=java-cross-compilation]"]
 include::sample[dir="snippets/java/crossCompilation/kotlin",files="build.gradle.kts[tags=java-cross-compilation]"]
 ====
 
+The only requirement is to have <<toolchains.adoc#sec:auto_detection,Java 7 installed and Gradle aware of it>>.
+
+You can take a look at what you needed to do with Gradle versions up to 6.6 https://docs.gradle.org/6.6.1/userguide/building_java_projects.html#compiling_and_testing_java_67[here].
```

Comment:
:x: It feels strange to have this in the docs. I'm fine with the _release notes_, but for the documentation itself I'd live this out.

Reply:
Ok, Piotr had a similar concern, removing.